.github/workflows/main.yml

```diff
@@ -37,18 +39,18 @@ jobs:
     changed-files:
         runs-on: ubuntu-latest
         outputs:
-            any_changed: ${{ steps.changed-files.outputs.any_changed }}
-            modified_files: ${{ steps.changed-files.outputs.modified_files }}
+            sol_any_changed: ${{ steps.changed-files.outputs.any_changed }}
+            sol_modified_files: ${{ steps.changed-files.outputs.modified_files }}
         steps:
             - uses: actions/checkout@v4
             - name: Get changed files
               id: changed-files
               uses: tj-actions/changed-files@v44
               with:
-                  files: packages/**/*.{sol,json,ts}
+                  files: packages/**/*.sol
```

Comment:
What happens if I update tests (TS) and open a PR?

Reply:
Right, I had forgotten the test are written in ts on `main`. I'll revert that change.
Otherwise only the style job would run if on ts files are changes